### PR TITLE
[CognitiveServices/BatchAvatar] Add API version 2024-08-01

### PR DESCRIPTION
**Pull request by zhifzhan**
# Data Plane API Specification Update Pull Request

> [!TIP]
> Overwhelmed by all this guidance? See the `Getting help` section at the bottom of this PR description.

No API schema change. Add new version 2024-08-01 for GA release.

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

spec_pr_review_workflow_diagram

## API Info: The Basics

Most of the information about your service should be captured in the issue that serves as your [*API Spec engagement record*](https://aka.ms/azsdk/onboarding/restapischedule).

* Link to API Spec engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [x] GA release

### Change Scope

<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous API Spec document (if applicable), and the root paths that have been updated. </sup>
* Design Document: No change since last preview version
* Previous API Spec Doc: No change since last preview version
* Updated paths: No change since last preview version

### Viewing API changes

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

### Suppressing failures

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[Swagger-Suppression-Process](https://aka.ms/pr-suppressions) 
to get approval.

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling

 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://github.com/Azure/azure-api-style-guide/blob/main/README.md)

### Guidelines & Specifications

 * [Azure REST API Guidelines](https://aka.ms/azapi/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/azapi/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links

 * [Schedule a data plane REST API spec review](https://aka.ms/azsdk/onboarding/restapischedule)

</details>

## Getting help

- First, please carefully read through this PR description, from top to bottom.
- If you don't have permissions to remove or add labels to the PR, request `write access` per [aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories](https://aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories) 
- To understand what you must do next to merge this PR, see the `Next Steps to Merge` comment. It will appear within few minutes of submitting this PR and will continue to be up-to-date with current PR state.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- If the PR CI checks appear to be stuck in `queued` state, please add a comment with contents `/azp run`.
  This should result in a new comment denoting a `PR validation pipeline` has started and the checks should be updated after few minutes.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json [hunk 1]
```diff
@@ -0,0 +1,848 @@
...
+        ],
+        "description": "Gets the list of batch syntheses.",
+        "operationId": "BatchSyntheses_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/QuerySkip"
+          },
```
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json:26]: Just a caution, that you are not going to remove this `skip` query param in future.

File: specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json [hunk 2]
```diff
@@ -0,0 +1,848 @@
...
+        "createdDateTime": {
+          "description": "The time-stamp when the object was created.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BatchSynthesisBillingDetails": {
+      "description": "Bath synthesis billing details.",
```
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json:349]: typo Bath

Please double check for other places.

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json (hunk 1 above)
Comment (zhifzhan) [reply specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json:26]: yes, we will have `skip` in all future versions

File: specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json (hunk 2 above)
Comment (zhifzhan) [reply specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json]: oops. :) thanks for reminding, fixed

**Review by yinhew — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
SDK is a required step. Also new spec need to be written as TypeSpec.
https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/launchcriterialong

Please make sure you

**Comment by zhifzhan** [comment]
> SDK is a required step. Also new spec need to be written as TypeSpec. https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/launchcriterialong
> 
> Please make sure you have a plan for SDK on your GA service.
> 
> SDK question aside, we still need one more approval from API Review Board for the sign-off of the GA API.

Internally discussed and we are planning an SDK once this GA version is online.
As for TypeSpec, there is another speech features from our team writing TypeSpec (VideoTranslation). We had a discussion and I want to start a separated PR to migrate Batch Avatar API to TypeSpec, after Speech's TypeSpec folder structure is determined.

**Review by MushMal — APPROVED** [review]
Looked at the changes which are renaming only. It's harder to reason about the consistency between this service spec and the rest of the family of the services. I am looking forward when the service will move to TypeSpec and will use a common library based on azure core which will standardize the specs.

**Comment by weidongxu-microsoft** [comment]
I've added `APIStewardshipBoard-SignedOff` as we have 2 approvals.

Service need to add `PublishToCustomers` to signal the API is published to customer (or at least ready to publish as it is).

**Comment by weidongxu-microsoft** [comment]
The "TypeSpec Requirement (data-plane)" required CI seems to be a new mandatory.

I can find the doc on suppression here https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-suppressions#suppressionsyaml
(I haven't done this or seen service done this, yet)

**Comment by zhifzhan** [comment]
ACK: ready to publish to customer.
(I don't have permission to add `PublishToCustomers` label by myself)

**Comment by weidongxu-microsoft** [comment]
> ACK: ready to publish to customer. (I don't have permission to add `PublishToCustomers` label by myself)

Added `PublishToCustomers` on behalf of service dev.

**Pull request merged**